src/plugins/intel_cpu/src/nodes/interpolate.cpp

```diff
@@ -1759,52 +2035,116 @@ void Interpolate::initSupportedPrimitiveDescriptors() {
+            if (shapeCalcMode == InterpolateShapeCalcMode::sizes) {
+                config.inConfs[SIZE_OR_SCALE_ID_V11].setMemDesc(
+                    creatorsMap.at(LayoutType::ncsp)->createSharedDesc(targetShapeType, getInputShapeAtPort(SIZE_OR_SCALE_ID_V11)));
+            } else {
+                config.inConfs[SIZE_OR_SCALE_ID_V11].setMemDesc(
+                    creatorsMap.at(LayoutType::ncsp)->createSharedDesc(scalesType, getInputShapeAtPort(SIZE_OR_SCALE_ID_V11)));
+            }
 
-        if (isAxesSpecified)
-            config.inConfs[AXES_ID].setMemDesc(creatorsMap.at(LayoutType::ncsp)->createSharedDesc(axesType, getInputShapeAtPort(AXES_ID)));
+            if (isAxesSpecified)
+                config.inConfs[AXES_ID_V11].setMemDesc(
+                    creatorsMap.at(LayoutType::ncsp)->createSharedDesc(axesType, getInputShapeAtPort(AXES_ID_V11)));
 
-        config.outConfs[0].setMemDesc(creatorsMap.at(dataFormat)->createSharedDesc(outputPrecision, getOutputShapeAtPort(0)));
+            config.outConfs[0].setMemDesc(creatorsMap.at(dataFormat)->createSharedDesc(outputPrecision, getOutputShapeAtPort(0)));
 
-        if (useAclExecutor) {
-            std::vector<MemoryDescPtr> srcMemoryDescs;
-            for (int i = 0; i < config.inConfs.size(); i++) {
-                srcMemoryDescs.push_back(config.inConfs[i].getMemDesc());
-            }
-            std::vector<MemoryDescPtr> dstMemoryDescs;
-            for (int i = 0; i < config.outConfs.size(); i++) {
-                dstMemoryDescs.push_back(config.outConfs[i].getMemDesc());
-            }
+            if (useAclExecutor) {
+                std::vector<MemoryDescPtr> srcMemoryDescs;
+                for (int i = 0; i < config.inConfs.size(); i++) {
+                    srcMemoryDescs.push_back(config.inConfs[i].getMemDesc());
+                }
+                std::vector<MemoryDescPtr> dstMemoryDescs;
+                for (int i = 0; i < config.outConfs.size(); i++) {
+                    dstMemoryDescs.push_back(config.outConfs[i].getMemDesc());
+                }
 
-            auto factory = std::make_shared<InterpolateExecutorFactory>(interpAttrs, srcMemoryDescs, dstMemoryDescs,
+                auto factory = std::make_shared<InterpolateExecutorFactory>(interpAttrs, srcMemoryDescs, dstMemoryDescs,
                                                                         std::make_shared<ExecutorContext>(context, getPrimitivesPriority()));
-            if (!factory->isEmpty()) {
-                supportedPrimitiveDescriptors.push_back({config, implDetail, factory});
+                if (!factory->isEmpty()) {
+                    supportedPrimitiveDescriptors.push_back({config, implDetail, factory});
+                }
+            } else {
+                supportedPrimitiveDescriptors.push_back({config, implDetail});
             }
+        };
+
+#if defined (OV_CPU_WITH_ACL)
+                interpAttrs.hasPad = hasPad;
```

Comment:
1, Please fix alignment 
2. pushDesc is mostly copypasted. Can it be extracted and reused for both branches?

Reply:
done, thank you!